pycbuf/CBufParserPy.cpp

```diff
@@ -424,7 +422,7 @@ static PyObject* DynamicStr(PyObject* obj) {
         break;
       }
       default:
-        VLOG_ASSERT(false, "Unsupported type: %d", md->type);
+        return nullptr;
```

Comment:
I agree the assert is strong, but now we lose logging or information about which unsupported type was there.